_sass/_components--technologies.scss

```diff
@@ -0,0 +1,57 @@
+$size--number-of-techs:      6;
+$size--tech-width-large:     100% / 6 - 1; // all techs on a single row
```

Comment:
that comment is great but why we use `-1`? I want to minimize my effort to understand the code while I am reading, we need to make it as clear and obvious as possible